crates/hc_sandbox/examples/setup_5_properties.rs

```diff
@@ -1,6 +1,6 @@
 use std::path::PathBuf;
 
-use hc_sandbox::calls::ActivateApp;
+/*use hc_sandbox::calls::ActivateApp;
```

Comment:
Remove these comments?

Reply:
freesig asked to keep them until this get's fixed